package.json

```diff
@@ -2,8 +2,7 @@
   "name": "convert-units",
   "version": "3.0.0-beta.0",
   "description": "Convert between quantities in different units",
-  "main": "./lib/cjs/index.js",
-  "module": "./lib/esm/index.js",
```

Comment:
Do we have to remove the `"module"` field? From what I remember, `"module"` can be used by bundlers like Webpack/Rollup in order to do better (??) bundling. [This](https://github.com/rollup/rollup/wiki/pkg.module) may help.

Reply:
Thanks for pointing this out! I meant to circle back and figure out what "module" does before committing my changes.